### DISCUSSION TITLE
Is there a way to use export (to Excel) feature from the frontend?

### DISCUSSION THREAD

**Discussion by fred-the-coder** [joomengine/Joomla-Component-Builder#689] [Q&A] [answered]
Hello,

I have built a component using JCB.
From the backend, I can use export to Excel feature.

I tried to set it in the coresponding front-end view. But I cannot.

Could someone helps me?

Thank you.

What I did: the component is dealing with register, thus I have views registers/register.
As the registers controller is not present in the frontend component, I tried to add the task "exportData" in frontend register, that I call from a button. 
But at this point starts the nightmare, xls method is not present in my component helper.
Thus I copied it...but again others "include" issues come...

**Comment by Llewellynvdm** [comment, marked as answer]
Okay here is a very powerful project to solve this very simply... https://datatables.net/examples/

I have used this on the front-end and it really saves you lots of pain... (if I could I would move Joomla back-end to this :)

**Reply by fred-the-coder** [reply]
OK, I saw this in your tutorial.
I thought it was Footable... I'll have a look.
Thank you for your reply.

**Reply by Llewellynvdm** [reply]
yes I started out with footable, and we still use it in the back-end of JCB with linked views... but I think we should move over to datatables... as they work with uikit and more..

**Comment by MastersOfMedia** [comment]
Dito here
